reflex/reflex.py

```diff
@@ -255,6 +257,51 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command(name="init")
+def db_init():
+    """Create database schema and migration configuration."""
+    if get_config().db_url is None:
+        console.print("[red]db_url is not configured, cannot initialize.")
+    if Path(constants.ALEMBIC_CONFIG).exists():
+        console.print(
+            "[red]Database is already initialized. Use "
+            "[bold]reflex db makemigrations[/bold] to create schema change "
+            "scripts and [bold]reflex db migrate[/bold] to apply migrations "
+            "to a new or existing database.",
+        )
+    prerequisites.get_app()
+    model.Model.alembic_init()
+    model.Model.migrate(autogenerate=True)
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
+    prerequisites.get_app()
+    if not prerequisites.check_db_initialized():
+        return
+    model.Model.migrate()
+    prerequisites.check_schema_up_to_date()
+
+
+@db_cli.command()
+def makemigrations(
```

Comment:
If I do the following, I get an error:

1. `reflex db init`
2. `rm reflex.db` (For example, if someone else clones the repo, they won't have the `.db` file)
3. `reflex db makemigrations` leads to `alembic.util.exc.CommandError: Target database is not up to date.`

If I run `reflex db migrate` first it will recreate the `.db` file. Maybe we should have a message if they try to run `makemigrations` without an existing db, telling them to run migrate first.